.github/workflows/action-deploy-apps.yml

```diff
@@ -219,3 +219,73 @@ jobs:
         if: ${{failure() || success()}}
         continue-on-error: true
         run: az logout
+
+  deploy-jobs:
+    name: Deploy jobs to ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    needs: [deploy-migration-job, deploy-apps]
+    environment: ${{ inputs.environment }}
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - name: resource-registry-syncer-job
```

Comment:
instead of having to manually insert the values in the matrixes for apps, migrations and jobs, we could rather have separate folders and use another job to extract all names and apply to matrix so all services are added automatically 🤔 (I'll add in a later PR)